spec/controllers/proposals_controller_spec.rb

```diff
@@ -194,6 +194,7 @@
       es_execute_with_retries 3 do
         get :download, text: "Work Order", format: "csv"
         expect(response.body).to include "Work Order 29"
+        expect(response.body).to include proposals.last.client_data.approving_official.display_name
```

Comment:
just so we're talking about the same thing: `user.display_name` returns `Jane Smith <jane@example.com>` if the first/last are defined, otherwise returns just `<jane@example.com>`.